src/test/datascience/jupyter/serverSelector.unit.test.ts

```diff
@@ -302,32 +298,16 @@ suite('DataScience - Jupyter Server URI Selector', () => {
     suite('Default Uri when selecting remote uri', () => {
         const defaultUri = 'https://hostname:8080/?token=849d61a414abafab97bc4aab1f3547755ddc232c2b8cb7fe';
 
-        async function testDefaultUri(expectedDefaultUri: string, clipboardValue?: string) {
+        async function testDefaultUri(expectedDefaultUri: string) {
             const showInputBox = sinon.spy(MultiStepInput.prototype, 'showInputBox');
             const { selector } = createDataScienceObject('$(server) Existing', 'http://localhost:1111', true);
-            when(clipboard.readText()).thenResolve(clipboardValue || '');
-
-            await selector.selectJupyterURI(true);
+            await selector.selectJupyterURI();
 
             assert.equal(showInputBox.firstCall.args[0].value, expectedDefaultUri);
         }
 
         test('Display default uri', async () => {
             await testDefaultUri(defaultUri);
         });
-        test('Display default uri if clipboard is empty', async () => {
```

Comment:
Not supporting the clipboard anymore. Fixes the problem with web where the user may not have clipboard permissions. 

New workflow doesn't really need clipboard as the user can also filter in the same entry so defaulting to the url in the clipboard doesn't really make sense.